packages/ui/v2/core/Meta.tsx

```diff
@@ -52,9 +52,10 @@ export default function Meta({ title, description, backButton, CTA }: MetaType)
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [title, description, backButton, CTA]);
 
+  const title_ = `${title} | Cal.com`;
```

Comment:
this change fixed this warning.

`
Warning: A title element received an array with more than 1 element as children. In browsers title Elements can only have Text Nodes as children. If the children being rendered output more than a single text node in aggregate the browser will display markup and comments as text in the title and hydration will likely fail and fall back to client rendering
`
for more context : https://github.com/vercel/next.js/discussions/38256

Reply:
TIL 🙌🏽

Reply:
New to me also 😀